src/App.css

```diff
@@ -0,0 +1,11 @@
+.conteiner {
+    padding: 0;
+    margin: 10px auto;
+  
+    max-width: 500px; 
```

Comment:
Please extend your app width to full window width, otherwise it'll become uncomfortable when you'll start to maintain any data.